src/openforms/scss/pdfs/_submission-step-row.scss

```diff
@@ -1,6 +1,9 @@
 @import '~microscope-sass/lib/typography';
 
 .submission-step-row {
+  // TODO this fixes overlap on next page but also adds empty pages in case of long values
+  break-inside: avoid-page;
```

Comment:
For some reason, the label of a field is duplicated if the page breaks in that row, that's why I added this
![Screenshot from 2024-07-02 15-08-10](https://github.com/open-formulieren/open-forms/assets/29249171/ef8235f3-3688-45ae-81ea-592a9b5d43be)

Unfortunately this does cause empty pages in case of really long field values (same problem mentioned here https://github.com/open-formulieren/open-forms/pull/4327#pullrequestreview-2076645221)

Reply:
definitely the lesser evil, so I'll accept it. it might be good to reserve some time at some point to check how we can overhaul/fix the PDF rendering and make it more robust